coordinator/src/test/scala/filodb.coordinator/queryplanner/PlannerHierarchySpec.scala

```diff
@@ -1198,58 +1198,23 @@ class PlannerHierarchySpec extends AnyFunSpec with Matchers with PlanValidationS
-          |-E~PromQlRemoteExec(PromQlQueryParams(sum(test{job="app"}) + 123,306,456,789,None,false), PlannerParams(filodb,None,None,None,None,30000,1000000,100000,100000,18000000,false,86400000,86400000,false,true,false,false), queryEndpoint=remote1-url, requestTimeoutMs=10000) on InProcessPlanDispatcher(QueryConfig(10 seconds,300000,1,50,antlr,true,true,Some(10000),None,true,false,true))""".stripMargin),
-      ("""sgn(test{job="app"}) + 123""",
-        """E~MultiPartitionDistConcatExec() on InProcessPlanDispatcher(QueryConfig(10 seconds,300000,1,50,antlr,true,true,Some(10000),None,true,false,true))
-          |-E~PromQlRemoteExec(PromQlQueryParams(sgn(test{job="app"}) + 123,123,456,306,None,false), PlannerParams(filodb,None,None,None,None,30000,1000000,100000,100000,18000000,false,86400000,86400000,false,true,false,false), queryEndpoint=remote0-url, requestTimeoutMs=10000) on InProcessPlanDispatcher(QueryConfig(10 seconds,300000,1,50,antlr,true,true,Some(10000),None,true,false,true))
-          |-E~PromQlRemoteExec(PromQlQueryParams(sgn(test{job="app"}) + 123,306,456,789,None,false), PlannerParams(filodb,None,None,None,None,30000,1000000,100000,100000,18000000,false,86400000,86400000,false,true,false,false), queryEndpoint=remote1-url, requestTimeoutMs=10000) on InProcessPlanDispatcher(QueryConfig(10 seconds,300000,1,50,antlr,true,true,Some(10000),None,true,false,true))""".stripMargin),
-      // BUG: Should sum without stitching
-      // ("""sum_over_time(test{job="app"}[5m])""",
-      //   """E~StitchRvsExec() on InProcessPlanDispatcher(QueryConfig(10 seconds,300000,1,50,antlr,true,true,Some(10000),None,true,false,true))
-      //     |-E~PromQlRemoteExec(PromQlQueryParams(sum_over_time(test{job="app"}[5m]),123,456,306,None,false), PlannerParams(filodb,None,None,None,None,30000,1000000,100000,100000,18000000,false,86400000,86400000,false,true,false,false), queryEndpoint=remote0-url, requestTimeoutMs=10000) on InProcessPlanDispatcher(QueryConfig(10 seconds,300000,1,50,antlr,true,true,Some(10000),None,true,false,true))
-      //     |-E~PromQlRemoteExec(PromQlQueryParams(sum_over_time(test{job="app"}[5m]),789,456,789,None,false), PlannerParams(filodb,None,None,None,None,30000,1000000,100000,100000,18000000,false,86400000,86400000,false,true,false,false), queryEndpoint=remote1-url, requestTimeoutMs=10000) on InProcessPlanDispatcher(QueryConfig(10 seconds,300000,1,50,antlr,true,true,Some(10000),None,true,false,true))""".stripMargin),
+    val stepSec = 45
+    val endSec = 6789
+    // All PromqlRemoteExecs should come in pairs (since there are two partitions), where the first
+    //   ends at 3306 (midpoint of 123 and 6789), and the second starts at 3633
+    //   (midpoint + lookback; snapped to periodic step).
+    val expectedQueryParams = Set(
+      ("remote0-url", TimeStepParams(startSec, stepSec, 3306)),
+      ("remote1-url", TimeStepParams(3633, stepSec, endSec))
+    )
+    val queries = Seq(
```

Comment:
TODO: add more tests to cover the changes to Aggregates/Subqueries/PeriodicSeriesWithWindowing